charts/kubewarden-controller/chart-values.yaml

```diff
@@ -110,3 +110,8 @@ auditScanner:
   containerRestartPolicy: Never
   # Additional namespaces that the audit scanner will not scan:
   skipAdditionalNamespaces: []
+  logLevel: info
+  # File path to CA cert in PEM format of PolicyServer endpoints
+  policyServerCa: "/pki/policy-server-root-ca-pem"
```

Comment:
I don't think we should offer this option. The certificate used by the Policy Server is generated by the Kubewarden controller, there's no way to provide a custom one. Hence we can have the "mount" point of the certificate hard coded